### PR TITLE
fix make clean logic

### PR DESCRIPTION
**Pull request by zhichen3**

### CODE REVIEW

**Review by zingale — APPROVED** [review]

**Comment by zingale** [comment]
instead of relying on `NET_TABLES`, we might just want to delete `*electroncapture.dat` and `*betadecay.dat`

**Comment by zhichen3** [comment]
> instead of relying on `NET_TABLES`, we might just want to delete `*electroncapture.dat` and `*betadecay.dat`

**Comment by zhichen3** [comment]
> instead of relying on `NET_TABLES`, we might just want to delete `*electroncapture.dat` and `*betadecay.dat`

updated

File: Make.Microphysics_extern [hunk 1]
```diff
@@ -148,5 +148,6 @@ endif
 clean::
 	@if [ -L helm_table.dat ]; then rm -f helm_table.dat; fi
 	@if [ -L reaclib_rate_metadata.dat ]; then rm -f reaclib_rate_metadata.dat; fi
-	@if [ ! -f nse19.tbl ]; then rm -f nse19.tbl; fi
-	$(foreach t, $(NET_TABLES), $(shell if [ ! -L `basename $t` ]; then rm -f `basename $t` ; fi))
+	@if [ -L nse.tbl ]; then rm -f nse.tbl; fi
+	$(foreach t, $(wildcard *_betadecay.dat), $(shell if [ -L $t ]; then rm -f $t; fi))
+	$(foreach t, $(wildcard *_electroncapture.dat), $(shell if [ -L $t ]; then rm -f $t; fi))
```
Comment (yut23) [inline Make.Microphysics_extern:153]: `wildcard` can take multiple arguments, so you don't have to repeat the second line.
```suggestion
	$(foreach t, $(wildcard *_betadecay.dat *_electroncapture.dat), $(shell if [ -L $t ]; then rm -f $t; fi))
```

**Pull request merged**